tools/IoTEdgeDevOps/DevOpsLib/BugPriorityExtension.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    using System.Collections.Generic;
+
+    public static class BugPriorityExtension
+    {
+        public static HashSet<BugPriority> BugPriorities => 
```

Comment:
I think this can be improved by using a single dictionary.

Dictionary<string, string> bugPriorityGroups = new Dictionary<string, string>{
    { "1", "Pri-1" },
    { "2", "Pri-2" },
    { "Other", "Pri-Other" }
}

For any usage of BugPriorities, use bugPriorityGroups.Keys.
For any usage of PriorityValue method, use key of dictionary in the loop.
For any usage of DisplayName method, use bugPriorityGroups[<key>].

Once simplified, remove this class and define this single dictionary in BugQueryGenerator